descriptors/ModuleDescriptor-template.json

```diff
@@ -683,6 +683,16 @@
             "usergroups.collection.get",
             "usergroups.item.get"
           ]
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/circulation/requests/instances",
+          "permissionsRequired": [
+            "circulation.requests.instances.item.post"
+          ],
+          "modulePermissions": []
```

Comment:
The implementation appears to make requests to other endpoints. Should the relevant permissions for these be declared here?

Reply:
Yes. I added this with my branch, but I didn't know what permissions would need to be added at that time. We'll need to figure this out so we can supply them here.

Reply:
Added the modulePermissions set